client/src/ui/organisms/article-actions/language-menu/index.scss

```diff
@@ -7,8 +7,44 @@
 }
 
 .language-menu {
+  li {
+    &:not(:first-child) {
+      padding-top: 1px;
+    }
+
+    &:not(:last-child) {
+      padding-bottom: 1px;
+    }
+  }
+
   .submenu-item {
-    padding: 0.5rem;
+    padding: 0.5rem !important;
+
+    &.locale-redirect-setting {
+      border-bottom: 1px solid var(--border-secondary) !important;
+      border-radius: 0 !important;
```

Comment:
Doesn't seem like this changes anything visually

Reply:
It does: The `border-bottom` adds a fine separator between the option and the languages.

And the `border-radius` prevents this separator to be curved at the edges.